library/public/image.h

```diff
@@ -27,33 +28,72 @@ class F3D_EXPORT image
     }
   };
 
-  image();
+  /**
+   * Create an image from file, the following formats are supported:
+   * PNG, PNM, TIFF, BMP, HDR, JPEG, GESigna, MetaImage, TGA.
+   * Can throw an image::exception in case of failure.
+   */
   explicit image(const std::string& path);
 
+  //@{
+  /**
+   * Default/copy/move constructors/operators
+   */
+  image();
   ~image();
-
   image(const image& img);
   image& operator=(const image& img);
-
   image(image&& img);
   image& operator=(image&& img);
+  //@}
+
+  //@{
+  /**
+   * Comparison operators, uses image::compare
```

Comment:
Add that it will return true if the images exactly match, otherwise we can use `compare` with a threshold